src/core/service.c

```diff
@@ -2882,9 +2980,20 @@ static int service_stop(Unit *u) {
 
 static int service_reload(Unit *u) {
         Service *s = ASSERT_PTR(SERVICE(u));
+        int r;
 
         assert(IN_SET(s->state, SERVICE_RUNNING, SERVICE_EXITED));
 
+        /* If we have extensions, try to reload them before we reload the
+         * service. For now, only reload confexts (instead of program code in
```

Comment:
It shouldn't say "for now", as we never intend to reload sysexts with this mechanism